internal/actions/runWorkflow.go

```diff
@@ -253,6 +254,11 @@ func finalize(inputs finalizeInputs) error {
 		}); err != nil {
 			return err
 		}
+
+		if pr != nil {
```

Comment:
I think you should also set this after line 61 if we have it

Reply:
I originally I had it there, but it seems like both codepaths would lead here eventually anyway (existing PR vs new PR). Stepping through `RunWorkflow()` it looks like it only returns errors prior to invoking `finalize()`, meaning it will always reach here in both cases (new vs existing PRs). Outside of errors of course.

Does that seem right? I can still add it, it just seems like it would be overwritten here anyway.

Reply:
We may not hit finalize if now changes are detected. But it would still be useful for us to know if a PR is already open for notifying about stale PRs

Reply:
oh that's a good point